### DISCUSSION TITLE
Few-shot learning idea

### DISCUSSION THREAD

**Discussion by SergheiDinu** [urchade/GLiNER#72] [Ideas]
Hi, 

I've seen that people are interested into how the current (v 2.1) model could use few-shot learning technique to increase performance. I have an Idea and I want to discuss it here. 
Let's we have to predict/label next sentence:  'Cristiano Ronaldo dos Santos Aveiro (Portuguese pronunciation: [kɾiʃˈtjɐnu ʁɔˈnaldu]; born 5 February 1985) is a Portuguese professional footballer who plays as a forward for and captains both Saudi Pro League club Al Nassr and the Portugal national team.'
On top of it, just like we do in another LLM:  'Lionel Andrés Messi is **footballer** Inter Miami **Soccer club** and the Argentina national team. **+** Cristiano Ronaldo dos Santos Aveiro (Portuguese pronunciation: [kɾiʃˈtjɐnu ʁɔˈnaldu]; born 5 February 1985) is a Portuguese professional footballer who plays as a forward for and captains both Saudi Pro League club Al Nassr and the Portugal national team.'

As you see above in first sentence/few-shot one, I presented that this is usually a name of footballer and soccer club. 

Best regards,
Serghei Dinu

**Comment by aayush-5** [comment]
Have you seen any improvement in the evaluation scores with this ?

**Comment by aayush-5** [comment]
@SergheiDinu can you show a code snippet how you can perform this few shot learning ?
